src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -949,6 +955,30 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         }
     }
 
+    // Method Description:
+    // - captures the pointer so that none of the other XAML elements respond to pointer events
+    // Arguments:
+    // - sender: XAML element that is interacting with pointer
+    // - args: pointer data (i.e.: mouse, touch)
+    void TermControl::_CapturePointer(Windows::Foundation::IInspectable const& sender, Windows::UI::Xaml::Input::PointerRoutedEventArgs const& args)
+    {
+        IUIElement uielem;
+        sender.as(uielem);

```

Comment:
What happens if the sender isn't a `IUIElement`?

Reply:
(I mean, I know that in our cases, this is fine, but I'd rather be more safe than sorry)

Reply:
I changed it to return a bool. It doesn't crash now but I could still totally see somebody using it improperly. Is this fine, though? Might be overthinking it.

---
In reply to: [293540403](https://github.com/microsoft/terminal/pull/1248#discussion_r293540403) [](ancestors = 293540403)